trello/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,37 @@
+= Trello Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Trello (Trello Connector) enables you to integrate custom applications into Trello to facilitate faster collaboration between teams.
+
+For information about compatibility and fixed issues, see the Trello Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Trello API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to:
+
+* The Trello target resource
+* Anypoint
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases For the Connector
+
+Use Trello Connector to manage Trello boards, lists, and cards.
```

Comment:
Customer Case Management
* Send customer case information managed by your support teams to your sales, development and other teams for better and faster collaboration .

Project Collaboration
* Manage key tasks for your project teams across all your IT apps (email, messaging, calendar) in one, streamlined and easy-to-use interface. Make working across teams smoother, especially in a remote work environment.

Reply:
@dhirsch1 added use cases.